src/main/java/donnafin/model/person/Notes.java

```diff
@@ -0,0 +1,39 @@
+package donnafin.model.person;
+
+/**
+ * Represents notes that are associated with the person in the address book.
+ *
+ */
+public class Notes implements Attribute {
+
+    public final String notes;
+
+    /**
+     * Constructs a note.
+     * @param note the note associated with the person.
+     */
+    public Notes(String note) {
+        this.notes = note;
```

Comment:
Do you think we can do this to keep the spelling consistent?

```suggestion
    public Notes(String notes) {
        this.notes = notes;
```